config-connector/solutions/networking/helm/mci/README.md

```diff
@@ -0,0 +1,153 @@
+# Multi-cluster ingress
+
+==================================================
+
+## NAME
+
+  multi-cluster ingress
+
+## SYNOPSIS
+
+Multi-cluster ingress solution uses network endpoint groups to provision load balancing across multiple clusters.
+This solution uses [gke-autoneg-controller](https://github.com/GoogleCloudPlatform/gke-autoneg-controller).
+For demonstration purposes it uses a docker image with a simple Node.js service (bulankou/node-hello-world) with a single endpoint that prints out a message.
```

Comment:
Any reason to use a custom image instead of `nginx`? (The latter is more likely to be available in clusters or easily retrieved).

Reply:
I would like an easy way to return a message on response, the message that I can specify in ENV parameters to the pod, not very familiar with nginx, how can I do it easily:
```
- name: node-app-container
        image: bulankou/node-hello-world
        env:
        - name: HELLO_MESSAGE
          value: {{ .Values.localMessage}}
        ports:
        - containerPort: 8080
```

Reply:
Ah, I missed that the first time around.

You could do this with nginx + configmap, but I agree it's probably not worth changing now.